src/PreflightTest.ts

```diff
@@ -930,104 +930,408 @@ export namespace PreflightTest {
+     */
     [Constants.PreflightRTCSampleBytesReceived]: number;
+
+    /**
+     * The bytes received at the time when the sample was taken.
+     */
     [Constants.PreflightRTCSampleBytesSent]: number;
+
+    /**
+     * The codec used by the underlying media connection.
+     */
     [Constants.PreflightRTCSampleCodec]: string;
+
+    /**
+     * The jitter present in the underlying media connection at the time when
+     * the sample was taken.
+     */
     [Constants.PreflightRTCSampleJitter]: number;
+
+    /**
+     * The evaluated MOS score of the underlying media connection at the time
+     * when the sample was taken.
+     */
     [Constants.PreflightRTCSampleMos]: number;
+
+    /**
+     * The number of packets lost during the `PreflightTest`.
+     */
     [Constants.PreflightRTCSamplePacketsLost]: number;
+
+    /**
+     * The fraction of total packets lost during the `PreflightTest`.
+     */
     [Constants.PreflightRTCSamplePacketsLostFraction]: number;
+
+    /**
+     * The number of packets received during the `PreflightTest`.
+     */
     [Constants.PreflightRTCSamplePacketsReceived]: number;
+
+    /**
+     * The number of packets sent during the `PreflightTest`.
+     */
     [Constants.PreflightRTCSamplePacketsSent]: number;
+
+    /**
+     * The round-trip time of a network packet at the time when the sample was
+     * taken.
+     */
     [Constants.PreflightRTCSampleRtt]: number;
+
+    /**
+     * The timestamp of when the RTC sample was taken during the
+     * `PreflightTest`.
+     */
     [Constants.PreflightRTCSampleTimestamp]: number;
   }
 
+  /**
+   * The call quality.
+   */
   export enum CallQuality {
+    /**
+     * Indicates `4.2 < average MOS`.
+     */
     Excellent = Constants.PreflightCallQualityExcellent,
+
+    /**
+     * Indicates `4.1 <= average MOS <= 4.2`.
+     */
     Great = Constants.PreflightCallQualityGreat,
+
+    /**
+     * Indicates `3.7 <= average MOS < 4.1`.
+     */
     Good = Constants.PreflightCallQualityGood,
+
+    /**
+     * Indicates `3.1 <= average MOS < 3.7`.
+     */
     Fair = Constants.PreflightCallQualityFair,
+
+    /**
+     * Indicates `average MOS < 3.1`.
+     */
     Degraded = Constants.PreflightCallQualityDegraded,
   }
 
+  /**
+   * The final report generated by the `PreflightTest` upon completion. Contains
+   * info related to the call quality and RTC statistics generated during the
+   * `PreflightTest`.
+   */
   export interface Report {
+    /**
+     * The `CallSid` of the underlying Twilio call used by the `PreflightTest`.
+     */
     [Constants.PreflightReportCallSid]: string;
+
+    /**
+     * The rated average MOS score of the `PreflightTest`. This value can help
+     * indicate the expected quality of future calls.
+     */
     [Constants.PreflightReportCallQuality]: CallQuality | null;
+
+    /**
+     * The Twilio Edge used for the `Call` underlying the `PreflightTest`.
```

Comment:
```suggestion
     * The Twilio Edge used for the `Call` in the `PreflightTest`.
```
My opinion, optional